zio-http/src/main/scala/zhttp/http/headers/HeaderGetters.scala

```diff
@@ -166,6 +166,11 @@ trait HeaderGetters[+A] { self =>
       Cookie.decodeRequestCookie(header)
     }
 
+  final def signedCookiesDecoded(secret: String): List[Cookie] =
+    headerValues(HeaderNames.cookie).flatMap { header =>
+      Cookie.decodeRequestCookie(header, Some(secret))
+    }
+
```

Comment:
I think we can overload `cookiesDecoded` with a `secret` for consistency.

Reply:
updated.